src/scip.rs

```diff
@@ -1315,13 +1331,7 @@ impl ScipPtr {
         let cons_is_transformed = unsafe { ffi::SCIPconsIsTransformed(cons.raw) } == 1;
         let var_is_transformed = unsafe { ffi::SCIPvarIsTransformed(var.raw) } == 1;
         let cons_ptr = if !cons_is_transformed && var_is_transformed {
-            let mut transformed_cons = MaybeUninit::<*mut ffi::SCIP_Cons>::uninit();
-            scip_call!(ffi::SCIPgetTransformedCons(
-                self.raw,
-                cons.raw,
-                transformed_cons.as_mut_ptr()
-            ));
-            unsafe { transformed_cons.assume_init() }
+            self.get_transformed_cons(cons)?.unwrap()
```

Comment:
@mmghannam In the removed code the result of  `getTransformedCons` is assumed to be non-null. Is this always correct or do we need to handle that case instead of unwrap?

Reply:
Thank you for noticing this. The (original) constraint can be deleted from presolving. So in case it's passed to this function it would cause undefined behavior. This should definitely be caught and I would suggest you add an expect with an error message "No transformed constraint was found for the passed original constraint, to prevent this you could disable presolving or mark the constraint to be modifiable"

Reply:
The same thing should be covered for the variable as it also can be deleted in presolving but this could be tackled in a following PR.